docs/UserGuide.md

```diff
@@ -421,12 +421,15 @@ Parameters:
 in the filtered contact list must have part of the `[term]` in the `[field]`
 specified above.
 * `[taken]` (optional). Only for filtering by `course`, this parameter additionally filters
-  out contacts who have finished the course.
+  out contacts who have finished/haven't started the course as of the current date of your system.
```

Comment:
Not sure why but this seems very confusing to me. `taken` refers to those who are currently taking the course?

Reply:
Yeah, I've thought about changing the term to 'taking', but does that count as a feature change?

Reply:
We have to keep it as it is, the best thing we can do now is to explain it clearly in the UG.

Reply:
ok